HS9_py2/heatsource9/ModelSetup/ModelSetup.py

```diff
@@ -1120,13 +1124,13 @@ def BuildZones_w_Values(self):
             # two angles so that late we can test whether we are between them, and only do the shading calculations
             # if that is true.
 
-            for i in xrange(radial_count): # Iterate through each transect direction
+            for i in range(radial_count): # Iterate through each transect direction
                 T_Full = () # lowest angle necessary for full sun
                 T_None = () # Highest angle necessary for full shade
                 W_Vdens_num = 0.0  #Numerator for the weighted Veg density calculation
                 W_Vdens_dem = 0.0  #Denominator for the weighted Veg density calculation
 
-                for s in xrange(transsample_count): # Iterate through each of the zones
+                for s in range(transsample_count): # Iterate through each of the zones
```

Comment:
I gotta say, I'm pretty unhappy with my code here. 1k line files is just really sloppy, and much of this code is untestable.